src/main/scala/vectorpipe/ProcessOSM.scala

```diff
@@ -250,14 +288,13 @@ object ProcessOSM {
       .select(
         lit(NodeType) as '_type,
         'id,
-        st_makePoint('lon, 'lat) as 'geom,
+        when('lon.isNotNull and 'lat.isNotNull, st_makePoint('lon, 'lat)) as 'geom,
```

Comment:
Are you finding places that these are null? They should be `Double.NaN`, but maybe that should be more consistent..

Reply:
Not sure I recall where this change propagated in from.  I might have based this PR on a working copy that wasn't quite up to date.  I can pull back this change.  No longer remembering if this was strictly needed or just a debugging check that was left in accidentally.

Reply:
Never mind. I did this; null values at this stage in the process are indicators that either data was redacted or this represents a deleted node.

Reply:
Nah, this might be needed. I was apparently responsible in https://github.com/azavea/osmesa/commit/e8e5652632d217bea466529314bfa8f4b5e3005d

I think I'd accepted that nulls were going to be valid in the DataFrame, but was struggling with the fact that auto-boxing was going to always turn them into `Double`s on the way out, which would never be null (and would be coerced to `0.0`). `Double.NaN` was a partial workaround for that, providing a valid `Double` that wasn't null but also wasn't 0.